.gitignore

```diff
@@ -36,3 +36,5 @@ testdata
 # Azurite temp files
 __*__
 __*__.json
+
+.idea
```

Comment:
This is a personal preference nothing to do with the project. It should go to your own `.gitignore` file.